src/components/SharedComponents/Map/SwitchMapTypeButton.tsx

```diff
@@ -48,12 +48,6 @@ const SwitchMapTypeButton = ( {
       )}
       style={DROP_SHADOW}
       accessibilityLabel={t( "Toggle-map-type" )}
-      accessibilityRole="button"
-      accessibilityState={
-        currentMapType === "standard"
-          ? t( "Standard--map-type" )
-          : t( "Satellite--map-type" )
-      }
```

Comment:
`accessibilityState` is not a prop on `INatIconButton`. `accessibilityState` doesn't take a translated string so I'm not sure what the original intention was here